cmd/traffic/cmd/manager/managerutil/systema.go

```diff
@@ -15,6 +16,10 @@ type SystemaCRUDClient interface {
 	a8rcloud.Closeable
 }
 
+func AgentIsProprietary(ctx context.Context) bool {
+	return strings.Contains(GetAgentImage(ctx), "datawire.io/telepresence-traffic-agent")
```

Comment:
I don't think this is correct. The image name is `ambassador-telepresence-agent`.